deploy/harvester-cloud-provider/templates/deployment.yaml

```diff
@@ -47,5 +47,6 @@ spec:
       {{- end }}
       volumes:
         - name: cloud-config
-          secret:
-            secretName: cloud-config
\ No newline at end of file
+          hostPath:
+            path: /root/cloud-config.yaml
+            type: FileOrCreate
```

Comment:
How is the file generated?

Reply:
We have not confirmed the location of the cloud-config. It depends on the RKE2. I just fill a path randomly. I will change it to a variable.

Reply:
Does it mean this change is not working at the moment? Is there a related rke2 issue?

Reply:
https://github.com/rancher/rancher/issues/33611